apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLoggerMiddleware.cs

```diff
@@ -27,15 +28,16 @@ namespace Google.Cloud.Diagnostics.AspNetCore
     public sealed class ErrorReportingExceptionLoggerMiddleware
```

Comment:
Oh, never mind. Now I think I understand. The `IExceptionLogger` interface is for anything that logs exceptions (but not necessary _reports_ them). So the `ErrorReportingExceptionLogger` is the exception logger which happens to do so by using error reporting. Ignore my naming comments.